### PR TITLE
[R2M] Add go1.4-bootstrap to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
The goal is to remove the unecessary compilation of go1.4 from the 
go-feedstock everytime we build a new version of go>1.4.

This also means that go1.4 is the only strict dependency on C/C++/Fortran.

### CODE REVIEW

File: recipes/go-bootstrap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    the convenience of garbage collection and the power of run-time reflection.
+    It's a fast, statically typed, compiled language that feels like a
+    dynamically typed, interpreted language.
+  doc_url: https://golang.org/doc
+  dev_url: https://go.googlesource.com/go
+
+extra:
+  recipe-maintainers:
+    - nehaljwani
+    - scopatz
```
Comment (sodre) [inline recipes/go-bootstrap/meta.yaml]: @nehaljwani and @scopatz are you still okay being co-maintainers of go1.4-bootstrap?

Comment (scopatz) [reply recipes/go-bootstrap/meta.yaml]: Yeah that is fine

**Comment by sodre** [comment]
@conda-forge/staged-recipes,  go1.4-bootstrap is ready for review/merge.

File: recipes/go-bootstrap/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/go-bootstrap/meta.yaml]: @nehaljwani - are you OK with still being on this list?

Comment (sodre) [reply recipes/go-bootstrap/meta.yaml]: I removed @nehaljwani for the moment, since I can add him back after the feedstock is created.

**Comment by sodre** [comment]
@scopatz this should be ready to merge as well.

**Pull request merged**